src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -58,12 +59,19 @@ public async Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext
             DocumentCollectionInfo documentCollectionLocation;
             DocumentCollectionInfo leaseCollectionLocation;
             ChangeFeedProcessorOptions processorOptions = BuildProcessorOptions(attribute);
+            
             processorOptions.StartFromBeginning = attribute.StartFromBeginning;
+
             if (attribute.MaxItemsPerInvocation > 0)
             {
                 processorOptions.MaxItemCount = attribute.MaxItemsPerInvocation;
             }
 
+            if (DateTime.TryParse(attribute.StartFrom, out var startFromDateTime))
```

Comment:
Can we provide a friendlier message if this does not parse? Other validations in the same file throw InvalidOperationException I believe

Reply:
Fixed